docs/strategy-customization.md

```diff
@@ -256,7 +256,26 @@ def populate_entry_trend(self, dataframe: DataFrame, metadata: dict) -> DataFram
             ['enter_short', 'enter_tag']] = (1, 'rsi_cross')
 
         return dataframe
-    ```
+```
+
+!!! Note
+    `enter_long` column must be set, even when your strategy is a shorting-only strategy. On other hand, enter_short is an optional column and don't need to be set if the strategy is a long-only strategy
+    
+```python
+    def populate_entry_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+        dataframe.loc[: ,'enter_long'] = 0
+
+        dataframe.loc[
+            (
+                (qtpylib.crossed_below(dataframe['rsi'], 70)) &  # Signal: RSI crosses below 70
+                (dataframe['tema'] > dataframe['bb_middleband']) &  # Guard
+                (dataframe['tema'] < dataframe['tema'].shift(1)) &  # Guard
+                (dataframe['volume'] > 0)  # Make sure Volume is not 0
+            ),
+            ['enter_short', 'enter_tag']] = (1, 'rsi_cross')
+
+        return dataframe
+```
```

Comment:
Instead of this - maybe add a small side-sentence above to `This method will also define a new column, "enter_long", ` - which might be something like `... (which is a mandatory column)`.

Alternatively add it to the "enter short trades" notes box.

Otherwise, we have 3 consecutive "note" boxes - which in my experience leads to people not reading it's content (like, not at all).